install-and-configure/advanced-configuration/resource-consumption.md

```diff
@@ -4,63 +4,61 @@ Kubecost can run on clusters with thousands of nodes when resource consumption i
 
 ![Memory Reduction Steps](/images/resource-consumption.png)
 
-## Secondary clusters: Disabling Cloud Assets and running Kubecost in Agent Mode/With ETL and caching disabled
+## Disable Cloud Assets and running Kubecost in Agent Mode/with ETL and caching disabled on secondary clusters
 
-* Cloud Assets for all accounts can be pulled in on your primary cluster by pointing Kubecost to one or more management accounts. You can disable Cloud Assets on secondary clusters by setting the following Helm value:
-  * `--set kubecostModel.etlCloudAsset=false`
-* Secondaries can be configured strictly as metric emitters to save memory.
-* Learn more about how to best configure secondary clusters [here](/install-and-configure/install/multi-cluster/secondary-clusters.md).
+{% hint style="info" %}
+Cloud Assets is an outdated concept in Kubecost which has been replaced by [CloudCost](/install-and-configure/install/cloud-integration/README.md#cloudcost) as of v1.106.
+{% endhint %}
+
+Cloud Assets for all accounts can be pulled in on your primary cluster by pointing Kubecost to one or more management accounts. Therefore, you can disable Cloud Assets on secondary clusters by setting the following Helm value:
+
+* `--set kubecostModel.etlCloudAsset=false`
+
+Secondary clusters can be configured strictly as metric emitters to save memory. Learn more about how to best configure them in our [Secondary Clusters Guide](/install-and-configure/install/multi-cluster/secondary-clusters.md).
```

Comment:
This section is less about Cloud Assets, and more about Secondary Clusters. I'd recommend stating here that for all Secondary/Agent Kubecost deployments, they should be configured to be strictly metric emitters (i.e. disable CloudCost), then provide a link to that Secondary Clusters Guide